Sources/Build/llbuild.swift

```diff
@@ -80,18 +83,24 @@ public struct LLBuildManifestGenerator {
         var targets = Targets()
 
         // Create commands for all target description in the plan.
-        for buildTarget in plan.targets {
-            switch buildTarget {
-            case .swift(let target):
-                targets.append(createSwiftCompileTarget(target), isTest: target.isTestTarget)
-            case .clang(let target):
-                targets.append(createClangCompileTarget(target), isTest: target.isTestTarget)
+        for (target, description) in plan.targetMap {
+            switch description {
+            case .swift(let description):
+                targets.append(createSwiftCompileTarget(description),
+                    isReachable: plan.graph.targets.contains(target),
```

Comment:
This will be slow. We can add a method in package graph which internally uses a set for fast lookup.